src/SOS/SOS.UnitTests/Scripts/DumpGen.script

```diff
@@ -44,10 +44,16 @@ VERIFY: ^<HEXVAL>\s+3\s+<DECVAL>\s+DotnetDumpCommands\.Program\+DumpSampleClass
 
 COMMAND: dumpgen gen2
 VERIFY: ^<HEXVAL>\s+5\s+<DECVAL>\s+DotnetDumpCommands\.Program\+DumpSampleClass
+!VERIFY: ^<HEXVAL>\s+1\s+10024\s+System.Byte\[\]
 
 COMMAND: dumpgen loh
 VERIFY: ^<HEXVAL>\s+1\s+<DECVAL>\s+DotnetDumpCommands\.Program\+DumpSampleClass\[\]
 
+IFDEF:MAJOR_RUNTIME_VERSION_GE_5
+COMMAND: dumpgen poh
+VERIFY: ^<HEXVAL>\s+1\s+10024\s+System.Byte\[\]
```

Comment:
Note to self: `10024` won't be correct for x86, so adjust the pattern here and above.
This isn't related to the CI failure in the other comment.